app/_hub/flash/set-dynamic-upstream-host/index.md

```diff
@@ -0,0 +1,123 @@
+---
+name: Set Dynamic Upstream Host
+publisher: Flash
+version: 1.0.0
+
+desc: Constructs the upstream hostname dynamically based on the incoming request parameters
+description: |
+  This plugin can be used to dynamically construct upstream hostname and port number based on the key identifier passed in the incoming request. If the same upstream api is deployed in different servers or data centers, then this plugin can form the hostname for the upstream api dynamically to route it to particular server or data center without making any change in Kong route or service.
+  This plugin extracts the key identifier from the incoming request and following are the supported parameters -
+  * Header param
+  * Query param
+  * Path param
+  * Body param
+  
+  
+support_url: https://github.com/anup-krai/kong-plugin-set-dynamic-target-host/issues
+source_url: https://github.com/anup-krai/kong-plugin-set-dynamic-target-host
+license_type: Apache-2.0 
+
+type: plugin
+categories:
+  - traffic-control
+
+kong_version_compatibility:
```

Comment:
Has the plugin been tested with all of these versions? If not, we should limit this compatibility matrix only to those versions tested

Reply:
Updated the tested versions for this plugin.